### PR TITLE
Changed front page + results page design

### PR DESCRIPTION
**Pull request by janelljchen**
i got rid of the basic.html file and instead added a new one called results.html. You can delete the basic.html.

### CODE REVIEW

File: testapp/views.py [hunk 1]
```diff
@@ -67,14 +67,14 @@ def index(request):
 
 def info(request):
   print("RESTAURANTS", request.POST)
-  city = request.POST['location']
+  city = request.POST['location'].title()
```
Comment (freddiev4) [inline testapp/views.py:70]: Why do we call `.title()` here?

File: testapp/views.py [hunk 2]
```diff
@@ -67,14 +67,14 @@ def index(request):
...
   term = request.POST['term']
   data = [city, term]
   #yelp_call() returns dictionary of restaurant and its information
   restaurants = yelp_call(term, city)
 
   save_user_request(data)
-  return render(request, 'basic.html', {'restaurants': restaurants})
+  return render(request, 'results.html', {'restaurants': restaurants, 'city': data[0], 'term':data[1]})
```
Comment (freddiev4) [inline testapp/views.py:77]: 👍

**Review by freddiev4 — COMMENTED** [review]
Some feedback below. Can you also post a screenshot/gif of the layout here in a GitHub comment?

File: testapp/views.py (hunk 1 above)
Comment (janelljchen) [reply testapp/views.py:70]: ahh that's just to make the first letter capital (like boston --> Boston)

**Comment by janelljchen** [comment]
image

**Comment by janelljchen** [comment]
image

**Review by freddiev4 — APPROVED** [review]

**Pull request merged**